### ISSUE TITLE
TTY Port Problems for IDE v > 1.6.1

### ISSUE THREAD

**Issue by acnebs** [arduino/Arduino#3580]
System: MBPr on OS X Yosemite, 10.10.4

I started out using version 1.6.5 of the Arduino IDE to upload programs to my Arduino Uno. Unfortunately, in the port menu, no tty.* ports were showing up - only cu.* . I am able to upload a sketch via the /dev/cu.* port, but only once (I'm assuming this is just the nature of the cu device). After the first sketch upload, avrdude throws a timeout whenever I try to upload thereafter (replugging the USB cable allows me to upload another sketch). Upon inspection of my machine, the TTY devices existed in /dev/, so I knew it wasn't my machine. I swapped out the cable and tried my other Arduino. No luck.

Then I tried downgrading to version 1.6.1 of the IDE. Specifically, I tried the version labeled "Java 7+". The TTY ports show up fine, and I am able to upload consequent sketches without unplugging/plugging back in the USB cable.

Then I tried versions 1.6.2, 1.6.3, and 1.6.4. None of them display the TTY devices.

In conclusion: there is a problem with the Arduino IDE for versions above 1.6.1 that prevents TTY ports from showing up on Mac OS X Yosemite, specifically on a MBPr (haven't tried any other type of mac).

**Comment by acnebs** [comment]
Update: 1.6.2 does not work either.

Last working version (in regards to displaying TTY ports) on Mac OS X Yosemite is 1.6.1.

**Comment by ffissore** [comment]
Are you using a genuine Arduino? Otherwise it could be a driver issue, unrelated to the IDE
